docs/simulators.rst

```diff
@@ -31,6 +31,19 @@ You can install it with:
 
 Scenic supports both 2D and 3D rendering modes for MetaDrive simulations.
 
+.. note::
```

Comment:
Let's move this to right after the code block (the line about 2D/3D rendering should come afterward).